Projects/Core/HGLogger/Sources/Logger.swift

```diff
@@ -6,3 +6,41 @@
 //
 
 import Foundation
+import os.log
+
+public final class Logger: Loggable {
+    private let subsystem: String
+    private let minLevel: LogLevelType
+
+    public init(
+        subsystem: String = Bundle.main.bundleIdentifier ?? "default",
+        minLevel: LogLevelType = .debug
+    ) {
+        self.subsystem = subsystem
+        self.minLevel = minLevel
+    }
+
+    public func log(
+        _ message: String,
+        level: LogLevelType,
```

Comment:
```suggestion
        level: LogLevelType = .debug
```
같이 디폴트하나주면 좀더 편하게 쓸수잇을거같아용

Reply:
넵 반영하겠습니당